docs/k6-jes-example/README.md

```diff
@@ -1,201 +1,5 @@
-
-export const options = {
-  thresholds: {
-    http_req_failed: ['rate<0.01'], // http errors should be less than 1%
-    http_req_duration: ['p(95)<500'], // 95% of requests should be below 500ms
-  },
-};
-
-export default function () {
-  http.get('https://test-api.k6.io/public/crocodiles/1/');
-}
-```
-
-This file would be used for K6 performance testing. 
-
-## Trigger Sequence
-
-Let's trigger the sequence using the command 
-
-```
-keptn trigger sequence --sequence testMyService --project k6-jes --service microserviceA --stage production
-```
-
-You can trigger the sequence from Keptn Bridge or using [Keptn API](https://keptn.sh/docs/0.19.x/reference/api/) too.
-
-### Success Trigger
-
-The Sequence has been successfully triggered can be seen by the log of `job-executor-service started` in the below image
-
-![Success Trigger](./images/success_trigger.png)
-
-### K6 Execution Logs
-
-The `microserviceA` service will execute with a `zero exit code`, therefore it will be finished successfully. We can view the logs given by the k6 run command.
-
-![K6 Pass](./images/k6_pass.png)
-
-## Failing K6 Test
-
-We'll see how the service fails when the K6 performance testing fails. In `k6_test.js` file, we'll change the threshold from `500ms` to `5ms`... Forcing it to fail!
-
-```js
-export const options = {
-  thresholds: {
-    http_req_failed: ['rate<0.01'], // http errors should be less than 1%
-    http_req_duration: ['p(95)<5'], // 95% of requests should be below 5ms... Force Fail :)
-  },
-};
-```
-
-We'll have to update the resources on ***Git Upstream***. You can do that manually in the `production` branch or by using this command
-
-```
-keptn add-resource --project k6-jes --service microserviceA --stage production --resource ./production/microserviceA/files/k6_test_fail.js --resourceUri files/k6_test.js
-```
-
-Here, the `k6_test_fail.js` will replace the `k6_test.js` in the Git repo.
-
-### Trigger Sequence Again
-
-Let's re-run the Service using the same command to trigger the sequence 
-
-```
-keptn trigger sequence --sequence testMyService --project k6-jes --service microserviceA --stage production
-```
-
-### K6 Fail Logs
-
-Due to the change in the K6 test script, now the service `microserviceA` will execute with a `non-zero exit code`, hence it will be finished as a failure. 
-
-![K6 Fail](./images/k6_fail.png)
-
-## Debug
-
-- Make sure the resources are in the correct location. You can use this repo as a reference - [jainammm/keptn-k6-jes-tutorial](https://github.com/jainammm/keptn-k6-jes-tutorial)
-- View the logs of Job Executor Service by using the command
-
-```
-kubectl -n keptn-jes logs deployment/job-executor-service -f job-executor-service
-```
-
-## Demo Link
-
-- The Demo Link to this tutorial can be found here - [YouTube](https://www.youtube.com/watch?v=MtjbvnDbhP8)
+On the k6 blog, check out the [tutorial to run k6 tests in Keptn](https://k6.io/blog/performance-testing-in-keptn-using-k6/).
```

Comment:
should we remove this content from the Readme? because the files required will be appropriately linked, which are not in the blog... we can reference with a link here only I guess

Reply:
If we preserve this Readme file, I think we should link to the tutorial. 

I changed the content of the sentence https://github.com/keptn-sandbox/k6-service/pull/27/commits/52d51461249b4348b4c49edbdc54c8a5f9937b44 - perhaps, that one fits better. Otherwise, I'll remove the entire paragraph. Just let me know.

Reply:
yes, but I feel that if someone is already in the repo it'll be better if they follow the Readme only because they have links to the files needed directly 
so we can keep the Readme... and link the tutorial in a separate paragraph itself in the main project Readme... Also, we can keep the references to previous tutorials in the same place pointing to Readme and not the blog post... only because the files that need to be added are already here users can directly look at them with the hyperlink provided in the readme (which are not there in blog)

Reply:
Hey @jainammm  - I don't fully catch your proposal. Perhaps, it is better if you continue this PR or create a new one that supersedes it.